src/layers.jl

```diff
@@ -91,28 +91,90 @@ end
 skip_identity(inplanes, outplanes, downsample) = skip_identity(inplanes, outplanes)
 
 """
-    addrelu(x, y)
+    mlpblock(planes, expansion_factor = 4, dropout = 0., dense = Dense)
+
+Feedforward block used in many vision transformer-like models.
+
+# Arguments
+- `planes`: Number of dimensions in the input and output.
+- `hidden_planes`: Number of dimensions in the intermediate layer.
+- `dropout`: Dropout rate.
+- `dense`: Type of dense layer to use in the feedforward block.
+- `activation`: Activation function to use.
+"""
+function mlpblock(planes, hidden_planes, dropout = 0., dense = Dense; activation = gelu)
```

Comment:
What would be used in place of Dense here? 

I wonder if it should be:
```
mlpblock(planes::Int, hidden_planes::Int, activation = gelu; dropout::Real = 0.0)
```

Reply:
Or `mlpblock(in, hidden, activation = gelu; dense = Dense, dropout = 0.)` if there is a reason for customizing it.